common/etc/nginx/templates/default.conf.template

```diff
@@ -326,7 +326,7 @@ server {
 
     location @trailslash {
         # 302 to request without slashes
-        rewrite ^ $scheme://$http_host$request_uri/ redirect;
+        rewrite ^ $scheme://$http_host$uri/$is_args$query_string redirect;
```

Comment:
**Annotation**:
Insert the trailing slash after the path and preserve the query string after it.